static/styles/settings.scss

```diff
@@ -1545,6 +1545,7 @@ body:not(.night-mode) #account-settings .custom_user_field .datepicker {
     pointer-events: all;
 }
 
+#id_realm_waiting_period_setting,
 #id_realm_create_stream_policy,
 #id_realm_invite_to_stream_policy,
 #id_realm_org_join_restrictions,
```

Comment:
We should probably restructure these to use a common CSS class; but probably not a blocker for merging :)

@pragatiagrawal31 @shubhamdhama FYI.